hw/bsp/samd21/boards/seeeduino_xiao/board.h

```diff
@@ -32,7 +32,7 @@
 #endif
 
 // LED
-#define LED_PIN               26
+#define LED_PIN               17
```

Comment:
From schematic, it seems like the led is active low. Could you also change the `LED_STATE_ON` to 0 as well.

Reply:
You're right, I'll do that.

Reply:
I've changed LED_STATE_ON to 0 and amended my last commit. I hope I've done it right, but if not I can raise another PR.

Reply:
That is ok, I don't really mind the git log history. The commit message and actual code changes are far more important.